### ISSUE TITLE
Symmetries of a C2v system inconsistent with PySCF 2.0 and NWChem

### ISSUE THREAD

**Issue by mkrompiec** [pyscf/pyscf#1705]
See [issue #1464. ](https://github.com/pyscf/pyscf/issues/1464). The snippet from that issue, on the current master branch, returns:
```
['A1' 'A1' 'B2' 'A1' 'B1' 'A1' 'B2']
['A1' 'A1' 'B2' 'A1' 'B1' 'A1' 'B2']
```
The output for the unshifted and shifted molecule is the same, but it is the "incorrect" output, i.e. what Harry got for the "unshifted" molecule and not consistent with NWChem (or PySCF 2.0). 

PR https://github.com/pyscf/pyscf/pull/1643 changes the behaviour to be inconsistent with previous versions of PySCF and with NWChem.

**Comment by mkrompiec** [comment]
@sunqm can you comment? Is the new behavior preferred, even though it is inconsistent with previous versions and with NWChem?

**Comment by sunqm** [comment]
The symmetry detection code in PySCF 2.0 and older versions incorrectly assigns the axes for C2v molecules. It was fixed in #1201. I don't know which convention NWChem follows. I'm sure the symmetry irreps in PySCF 2.1 and newer versions are consistent with the C2v character tables from various sources, such as https://iopenshell.usc.edu/resources/howto/symmetry/ , http://symmetry.jacobs-university.de/cgi-bin/group.cgi?group=402&option=4 .  See also the discussion in the issue #1430 .

I'll mark this issue as solved for the moment. Feel free to reopen the issue to continue the discussion
